client/a8c-for-agencies/sections/signup/signup-v2/components/step-progress/style.scss

```diff
@@ -0,0 +1,73 @@
+.step-progress {
+	width: 100%;
+	background: var(--studio-white);
```

Comment:
We should probably avoid directly referencing `--studio-*` but instead use the equivalent theme variable. `--color-surface`